.github/workflows/auto-format.yml

```diff
@@ -14,10 +14,10 @@ jobs:
 
     # only run on pull requests so long as they don't come from forks
     if: ${{ !( (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name != github.repository) ) }}
-    uses: cloudposse/github-action-auto-format/.github/workflows/auto-format-reusable.yml@review-branch
+    uses: cloudposse/github-action-auto-format/.github/workflows/auto-format-reusable.yml@main
     with:
       # Delete "format-tasks:" values as desired to limit the scope of the overall auto-formatting task.
-      format-tasks: "[\"readme\", \"github\", \"terraform\"]"
+      format-tasks: "[\"readme\", \"github\", \"terraform\", \"context\"]"
```

Comment:
```suggestion
      format-tasks: |
        ["readme", "github", "terraform", "context"]
```

Reply:
This is done in JSON because we're using a matrix strategy in auto-format-reusable.yml and the argument to matrix has to be JSON-formatted. It's easiest the current way.

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy